Cargo.toml

```diff
@@ -1,16 +1,15 @@
 [package]
 name = "bevy_slippy_tiles"
-version = "0.4.3"
+version = "0.5.0"
 edition = "2021"
 description = "Provides slippy tile fetching functionality in the Bevy game engine"
 license = "MIT OR Apache-2.0"
 repository = "https://github.com/edouardpoitras/bevy_slippy_tiles"
 
 [dependencies]
-bevy = "0.12"
-futures-lite = "1.13"
+bevy = "0.13"
 googleprojection = "1.2"
-reqwest = { version = "0.11", features = ["blocking"] }
+reqwest = { version = "0.12", features = ["blocking"] }
```

Comment:
I've also noticed that reqwest has a newer version available. I included this update as well, but if you prefer not to merge it, you can easily revert commit d34b24f3d74ecf0b3420dc6c302c740311cb325e.